js_modules/dagster-ui/packages/ui-core/src/instigation/TickHistory.tsx

```diff
@@ -349,7 +349,7 @@ export const TickHistoryTimeline = ({
   const {ticks = []} = data.instigationStateOrError;
 
   const onTickClick = (tick?: InstigationTick) => {
-    setSelectedTickId(tick ? Number(tick.id) : undefined);
+    setSelectedTickId(tick ? Number(tick.tickId) : undefined);
```

Comment:
I believe this should fix the existing bug where clicking on a tick in the timeline is not properly opening the dialog.